BioProgSV/addscript/fastq_qc_function.py

```diff
@@ -0,0 +1,34 @@
+def fast_qc(seqs):
```

Comment:
Возможно, вы забыли аннотоции типов
```suggestion
def fast_qc(seqs: str) -> dict:
```